backend/src/useCases/ownership.ts

```diff
@@ -13,8 +13,18 @@ const setObjectAsDeleted = async (user: User, cid: string) => {
   await ownershipRepository.setObjectAsDeleted(user.provider, user.id, cid);
 };
 
+const getOwners = async (cid: string) => {
+  return ownershipRepository.getOwners(cid);
```

Comment:
I'm not sure that I understand why these just forward to a other function with the same name?
Why not just using getAdmins from ownershipRepository where you need it?

Furthermore, declaring the same function name in two place in the same project make it a bit harder to search across the codebase and understand the flow

Reply:
It's true that I could just call `ownershipRepository.getOwners(cid)` but I expected this function to get more complex in few iterations, e.g Once we have `users` table not returning `Ownership` entries. 

Generally speaking I prefer adding the two layers (repository and use case) because usually the use case functions get more complex within time and just from personal taste I prefer not to call a `B` repository in the context of entity `A`